debs/SPECS/3.11.0/wazuh-agent/debian/postinst

```diff
@@ -138,7 +138,7 @@ case "$1" in
 
     # Service
     if [ -f /etc/init.d/wazuh-agent ]; then
-        if [ -d /run/systemd/system ]; then
+        if [ ! -z "$(file /sbin/init |grep "systemd")" ]; then
```

Comment:
Question: what about to use: 
`        if [ ! -z "$(file /sbin/init |grep 'systemd')" ]; then`
instead?

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/269/commits/6765088880328ba825528bf1164578088b3bf8fa